src/pybind/mgr/dashboard/controllers/cephfs.py

```diff
@@ -101,6 +101,21 @@ def rename(self, name: str, new_name: str):
                 component='cephfs')
         return f'Volume {name} renamed successfully to {new_name}'
 
+    @UpdatePermission
+    @Endpoint('PUT')
```

Comment:
We should add a decorator `@EndpointDoc` to say what these params are and what is the expected input format, especially for vague fields like `cliend_id`.

Reply:
I understood recently that @EndpointDoc is something we should add if we need to have a clear [openapi doc](https://docs.ceph.com/en/latest/mgr/ceph_api/)

Reply:
Yep, added!